milestone-2.md

```diff
@@ -0,0 +1,25 @@
+#### Milestone 2
+
+<b>Audience:</b> Mainstream Consumers.
+<b>Overview:</b> Your self-described luddite friend finds Peerlio from a web search, after a brief chat with the Peerfolio Agent, Friend is convinced that they will use Peerfolio as their first venture into acquiring crypto.
+
+
+#### Major enhancements:
+- Product Design for multiplayer mode
```

Comment:
include a map of the crypto-native UX vs mainstream experience, and how far we want to go with the former.